apps/api/src/persistence/repositories/data-importer.ts

```diff
@@ -58,9 +58,11 @@ export class DataImporter {
         );
 
         await Promise.all(
-            Object.values(ArangoCollectionId).map((collectionName) =>
-                this.databaseProvider.getDBInstance().deleteAll(collectionName)
-            )
+            Object.values(ArangoCollectionId)
```

Comment:
Ah it'll be good to use dynamic registration for collection names. Also, we probably can do this in a single query.

Reply:
These are future concerns by the way.